app/packs/src/apps/mydb/elements/details/cellLines/analysesTab/EditModeRow.js

```diff
@@ -1,42 +1,39 @@
-import React, { Component } from 'react';
-import EditModeHeader from 'src/apps/mydb/elements/details/cellLines/analysesTab/EditModeHeader';
+import React from 'react';
+import Header from 'src/apps/mydb/elements/details/cellLines/analysesTab/Header';
 import ContainerComponent from 'src/components/container/ContainerComponent';
 import PropTypes from 'prop-types';
-import Panel from 'src/components/legacyBootstrap/Panel'
+import { Accordion } from 'react-bootstrap';
 
-// eslint-disable-next-line react/prefer-stateless-function
-export default class EditModeRow extends Component {
-  
-  render() {
-    const { container, parent, element,readOnly } = this.props;
-    return (
-      <Panel
-        eventKey={container.id}
-        key={container.id}
-      >
-        <Panel.Heading
-          onClick={() => parent.handleClickOnPanelHeader(container.id)}
-        >
-          <EditModeHeader 
-            element={element} 
-            container={container}
-            parent={parent} 
-            readOnly={readOnly}/>
-        </Panel.Heading>
-        <Panel.Body collapsible>
-          <ContainerComponent
-            analysisMethodTitle="Type (BioAssay Ontology)"
-            ontologyName="bao"
-            templateType="researchPlan"
-            readOnly={readOnly}
-            disabled={false}
-            container={container}
-            onChange={() => parent.handleChange(container)}
-          />
-        </Panel.Body>
-      </Panel>
-    );
-  }
+export default function EditModeRow({
```

Comment:
If you change this to a functional component.
Prefer to write it like this:
`export default EditModeRow  = ({ container, parent, element, readOnly }) => {`
or you can write it like this:
`const  default EditModeRow  = ({ container, parent, element, readOnly }) => {`
and the `export default EditModeRow` at the end of the file